packages/notifi-react-card/lib/components/AlertHistory/AlertDetailsCard.tsx

```diff
@@ -16,10 +18,21 @@ export const AlertDetailsCard: React.FC<AlertDetailsProps> = ({
   notificationEntry,
   classNames,
 }) => {
-  const detailsContents = useMemo(
-    () => getAlertDetailsContents(notificationEntry),
-    [notificationEntry],
-  );
+  const { bottomContent, otherContent, topContent, bottomContentHtml } =
+    useMemo(
+      () => getAlertDetailsContents(notificationEntry),
+      [notificationEntry],
+    );
+
+  const { sanitizedMarkdownBottomContent, sanitizedBottomContentHtml } =
+    useMemo(() => {
+      const parsedMarkdown = marked.parse(bottomContent);
```

Comment:
for now let's skip parsing the `bottomContent` -- we don't know what behaviors this might break since we were previously using it as just the string content to display

Reply:
Reverted back 🚀 CC: @juni-notifi 
1. Revert back not to parse `roadcastMessageEventDetails` 
2. Remove `marked` from dependency list